### ISSUE TITLE
Build / Run Failed

### ISSUE THREAD

**Issue by egtalbot** [MicrosoftDocs/visualstudio-docs#8970]
[Enter feedback here]
After following all the steps, I got this when tried to build or run
Severity	Code	Description	Project	File	Line	Suppression State
Error		The command "npm run clean" exited with code 1.	NodejsWebApp1	C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VisualStudio\v17.0\Node.js Tools\Microsoft.NodejsToolsV2.targets	56	

Perhaps I missed something, but it seems very logical that some people would get errors here so the page needs to have steps to troubleshoot that.
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c4b0ea28-4a31-a3ea-fa5f-c32d5c72d4bd
* Version Independent ID: 31bd064b-51eb-5505-986e-2c7148784085
* Content: [Create a Node.js and React app - Visual Studio (Windows)](https://learn.microsoft.com/en-us/visualstudio/javascript/tutorial-nodejs-with-react-and-jsx?view=vs-2022)
* Content Source: [docs/javascript/tutorial-nodejs-with-react-and-jsx.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/javascript/tutorial-nodejs-with-react-and-jsx.md)
* Product: **visual-studio-windows**
* Technology: **vs-javascript**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by egtalbot** [comment]
So it turns out the package.json file needed the following added:
```
 "scripts": {
    "build": "tsc --build",
    "clean": "tsc --build --clean"
  }
```
I'd say this should be added to the instructions on the page, because presumably you want users to utilize the Clean feature of VS. After doing this, it compiled and ran BUT there was no text on the html page. The instructions say I should see something about Welcome to React. This suggests that something is going wrong with the rendering of app.tsx. But there are no errors in VS or in Developer tools in the browser.

So once again I am stuck despite following the steps in the tutorial. Problems this basic are either not supposed to happen or be anticipated by the tutorial. So these instructions are pretty disappointing - I had another problem with an error when I pasted the code into app.tsx but I figured that one out - it wasn't picking up the jsx: react in the tsconfig for some reason even when i closed and opened VS. I had to use the hints to add it a second time, which generated a different error and then when I removed the duplicate everything started working.

**Comment by Mikejo5000** [comment]
@egtalbot Thanks for the feedback! I installed the latest version of Node.js and the latest version of the npm packages and retested. I found I needed to correct the webpack command line instruction (removing the "./app.tsx" portion on the command line), and then everything worked correctly, per instructions. I am updating the article with the updated information, which should be live by EOD 3/24/23.
The various steps in the article have a webpack dependency, and your build script instructions don't appear to use webpack, so that appears to be a slightly different scenario than we're targeting in the article. We're not planning to cover the variation at this time. Closing this for now.
